contribute.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "django-browserid",
+    "description": "django-browserid is a library that integrates BrowserID authentication into Django.",
```

Comment:
@Osmose I know it's the name of the package, but is `BrowserID` still a valid term? Should the description use the term `Persona` instead?